Bluepill.podspec

```diff
@@ -7,4 +7,9 @@ Pod::Spec.new do |s|
     s.author         = 'LinkedIn Corporation'
     s.source         = { http: "#{s.homepage}/releases/download/v#{s.version}/Bluepill-v#{s.version}.zip" }
     s.preserve_paths = '*'
+    s.prepare_command = <<-CMD
+                        ls -l
+                        cp -r Bluepill-v#{s.version}/* ./
```

Comment:
Instead of `cp -r` followed by `rm -rf`, why not `mv Bluepill-v#{s.version}/* ./`?